package.json

```diff
@@ -54,6 +54,7 @@
     "electron-builder-squirrel-windows": "~20.14.0",
     "electron-connect": "^0.6.3",
     "eslint": "^4.19.1",
+    "eslint-plugin-eslint-comments": "^3.1.2",
```

Comment:
I assume this is copied over from master, but this is requiring a newer version than master if that matters

Reply:
Good question, didn't go to Master for this. Just installed it on this branch because it was for some reason needing it, even though it seemingly hadn't previously? I will make sure master version matches! Good eye! 👁